src/types/index.tsx

```diff
@@ -9,7 +9,7 @@ export interface Member {
   status: UserStatus;
 }
 
-export interface BreadcrumbLinkType {
+export interface BreadcrumbsTypes {
```

Comment:
@nadirhussain786  name is not suitable for `interface`  , or you can export as type instead of `interface`